src/predefinedComponents/TabbedHeader/TabbedHeader.js

```diff
@@ -104,17 +80,12 @@ export default class TabbedHeader extends React.Component {
     })
 
     const renderImage = () => {
-      const logo = isUndefined(foregroundImage)
-      ? require('../../assets/images/photosPortraitMe.png')
-      : foregroundImage
+      const logo = isUndefined(foregroundImage) ? require('../../assets/images/photosPortraitMe.png') : foregroundImage
```

Comment:
why is this line that long? It has 117 characters, and I think it should be max 100 characters.